services/dumper.js

```diff
@@ -128,8 +129,11 @@ function Dumper(config) {
   }
 
   function writeModel(table, fields, references, options = {}) {
-    const templatePath = `${__dirname}/../templates/app/models/model.txt`;
-    const template = _.template(fs.readFileSync(templatePath, 'utf-8'));
+    const templatePath = config.dbDialect === 'mongodb' ?
+      `${__dirname}/../templates/app/models/mongo-model.hbs`
+      :
+      `${__dirname}/../templates/app/models/sequelize-model.hbs`;
```

Comment:
We never use this syntax. The linter is not correctly set on this project.
It should have been 
```
? .....
: ....;
```